cmake/Codegen.cmake

```diff
@@ -268,13 +304,16 @@ if(INTERN_BUILD_ATEN_OPS)
     )
   endforeach()
 
+  if(NOT USE_LIGHTWEIGHT_DISPATCH)
+    set(generated_unboxing_sources "")
+  endif()
```

Comment:
Could we add else, to empty dispatching sources with TORCH_LIBRARY? We could generate both, but would the compiler time be longer?

Reply:
Cool I think I can merge this to line 221